pkg/bgpv1/manager/reconcilerv2/neighbor.go

```diff
@@ -66,14 +68,11 @@ type PeerData struct {
 }
 
 // NeighborReconcilerMetadata keeps a map of running peers to peer configuration.
-// key is the peer name.
+// Key is the instance name.
```

Comment:
Hmm, strange I think I might have made a mistake. Intention was probably to store map of PeerData, with key as neighbor ID. Upsert metadata would be simple map update instead of iterating each neighbor.

Reply:
OK, I see. I pushed an additional commit which cleans this up (https://github.com/cilium/cilium/pull/34426/commits/0bba100fd67ec1e32309a416bc66c5b3c6f302f6)